src/vs/workbench/contrib/snippets/browser/commands/configureSnippets.ts

```diff
@@ -182,16 +182,32 @@ async function createSnippetFile(scope: string, defaultPath: URI, quickInputServ
 		'\t// used to trigger the snippet and the body will be expanded and inserted. Possible variables are: ',
 		'\t// $1, $2 for tab stops, $0 for the final cursor position, and ${1:label}, ${2:another} for placeholders. ',
 		'\t// Placeholders with the same ids are connected.',
-		'\t// Example:',
-		'\t// "Print to console": {',
-		'\t// \t"scope": "javascript,typescript",',
-		'\t// \t"prefix": "log",',
+		// --- Start Positron ---
+		'\t// Example snippet for R:',
+		'\t// "switch": {',
```

Comment:
`switch()` is one of the previously-built-in R snippets, which is why I chose it. I also wanted to pick something that seemed amenable to parallel R and Python examples. If I have failed, on the Python side, we can take another crack at picking an example that feels good for both worlds.